bson/src/test/unit/org/bson/BsonHelper.java

```diff
@@ -67,6 +67,39 @@
                 new BsonDocument("a", new BsonInt32(1)));
     }
 
+    private static List<BsonValue> getRelaxedBsonValues() {
```

Comment:
Painful but ensures we actually test the round tripping correctly for relaxed modes